eng/automation/generate_utils.py

```diff
@@ -414,7 +415,13 @@ def generate_typespec_project(
                 tsp_dir,
             ]
 
+            if disable_customization:
+                remove_customization_config(tsp_dir)
```

Comment:
We are unable to "unset" an emitter option via tsp-client (nor tsp compile).

Hence we had to modify the tspconfig.yaml

Reply:
Yes, I tried that. It can "set" a new value e.g. `NewClass.java`. But it cannot set it as if we didn't set the `customization-class` in tspconfig.yaml (e.g. `--emitter-option="customization-class="` won't work)